src/main/java/seedu/address/logic/commands/AddCommissionCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DEADLINE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FEE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TITLE;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.commission.Commission;
+
+/**
+ * Adds a commission to the selected customer.
+ */
+public class AddCommissionCommand extends Command {
+    public static final String COMMAND_WORD = "addcom";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a commission to the address book. "
+            + "Parameters: "
+            + PREFIX_TITLE + "TITLE "
+            + PREFIX_FEE + "FEE "
+            + PREFIX_DEADLINE + "DEADLINE "
+            + "[" + PREFIX_DESCRIPTION + "DESCRIPTION] "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_TITLE + "Tokyo Ghoul Kaneki "
+            + PREFIX_FEE + "50 "
+            + PREFIX_DEADLINE + "johnd@example.com "
```

Comment:
Can you update the command usage message so we know what parameters the `addcom` command accepts? Thanks.

Reply:
yes please update it :P 

`addcom n/test f/50 d/2022-02-02 s/true p/desc t/digital t/neon palette`

but tag regex checks currently don't allow spaces. we would need to enable it for the second tag

Reply:
Kk soz missed it